vizro-ai/src/vizro_ai/dashboard/nodes/imports_builder.py

```diff
@@ -0,0 +1,85 @@
+"""Import statement builder node."""
+
+from typing import List
+
+from langchain_core.prompts import ChatPromptTemplate
+
+try:
+    from pydantic.v1 import BaseModel, Field
+except ImportError:  # pragma: no cov
+    from pydantic import BaseModel, Field
+
+model_sum_prompt = ChatPromptTemplate.from_messages(
+    [
+        (
+            "system",
+            """You are a data assistant with expertise dashboard design and a visualization library named Vizro. \n
+            Summarize the user \n
+            question and list out Vizro models required for creating the dashboard. \n
+            Available Vizro models to choose from: \n ------- \n
+            [
+            'AgGrid' # for table creation,
+            'Card',
+            'Filter',
+            'Graph',
+            'Layout',
+            ]  \n ------- \n
+            Here is the user question:""",
+        ),
+        ("placeholder", "{messages}"),
+    ]
+)
+
+
+class ModelName(BaseModel):
+    """Vizro model name output."""
+
+    model_required: str = Field(description="Name of the Vizro model required")
+
+
+class ModelSummary(BaseModel):
+    """Vizro model summary output."""
+
+    model_summary: List[ModelName]
+
+
+def _generate_import_statement(models) -> str:
```

Comment:
Probably rope/jedi could do something of this automatically but not important if it works ok already

Reply:
Yes, and again, I think all of this should be done without LLM for sure!

Reply:
And then ideally we run `black` on it afterwards 🙂